src/core/compile/CompilationProcessor.ts

```diff
@@ -258,17 +258,11 @@ export class CompilationProcessor {
   }
 
   private _getUpdatedArtifactFileTypes(): ArtifactsFileType[] {
-    const fileTypes: ArtifactsFileType[] = [];
-
-    this._config.compileFlags.wasm && fileTypes.push("wasm");
-    this._config.compileFlags.c && fileTypes.push("c");
-    this._config.compileFlags.r1cs && fileTypes.push("r1cs");
-    this._config.compileFlags.sym && fileTypes.push("sym");
-    this._config.compileFlags.json && fileTypes.push("json");
-
-    return fileTypes;
+    const validFileTypes: Set<string> = new Set(["wasm", "c", "r1cs", "sym", "json"]);
```

Comment:
Could you also add a new line after this line and after the `_getUpdatedArtifactFileTypes` function?

Reply:
You still need to add a new line after `validFileTypes` variable